report/main.Rmd

```diff
@@ -0,0 +1,313 @@
+
+```{r sales-time-series-plot, fig.cap="This plot illustrates the daily sales volumes of zero-calorie (in green) and sugary (in blue) beverages across three hospital sites over 30 weeks. The dashed and their corresponding shorthand labels mark the interventions to allow for visual assessment of their impact on beverage sales trends.", echo=FALSE, fig.height=3, fig.width=6, fig.pos="H"}
+plot_sales_time_series(beverage_sales)
+```
+
+
+For more visual exploration see appendix.
+
+## 4. Formal Analysis
+
+Overall, three models are considered for formal analysis.
+
+```{r Handle Missing Data, echo=FALSE, message=FALSE, warning=FALSE}
+# handle_missing_data(beverage_sales)$MissingOverview
+```
+
+### 4.1 Interupted Times Series Analysis 
+
+The study's design, an interrupted time series across multiple sites, naturally lends itself to Interrupted Time Series Analysis (ITSA). This method effectively handles the challenges posed by non-randomized designs, isolating intervention impacts from pre-existing trends.
+
+The ITSA can characterize the immediate and ongoing effects of visual calorie content displays and price discounts on beverage selection. It segments the data across different intervention phases to quantify changes in zero-calorie and sugary beverage sales, offering a detailed view of each strategy's effectiveness over time.
+
+Moreover, ITSA's adaptability enables the analysis of variations in intervention effects across sites, enabling site-specific nuances. This approach reveals whether an intervention's success is uniform or site-dependent but also compares the impact of combined interventions against singular strategies. Therefore, employing ITSA can address the study's core questions, demonstrating its suitability for unravelling the effects of multifaceted interventions on consumer behaviour.
+
+```{r itsa, echo=FALSE}
+
+```
+
+
+
+### 4.2 Generalized Estimating Equations
+
+The Generalized Estimating Equations (GEE) approach is a convenient and relatively easy to interpret method to model longitudinal data. GEE is suitable for analyzing the data from this study since daily sales of bottled sugared beverages and zero-calorie beverages were measured repeatedly over time. GEE can be thought of as an extension of the Generalized Linear Model to longitudinal data (Columbia University). This method is particularly convenient due to its high statistical power, built-in handling of missing at random data, and its ability to account for within-subject correlation in non-normal data. 
```

Comment:
A brief introduction to what GEE is could be beneficial at the beginning by directly stating "GEE is a model that ..."